MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -535,7 +535,7 @@ protected override void InternalUpdate()
 			}
 			catch (Exception e)
 			{
-				App.Logger.LogError($"Failed to synchronize app.  Exception: {e.Message}\nStackTrace: {e.StackTrace}");
+				App?.Logger.LogError($"Failed to synchronize app.  Exception: {e.Message}\nStackTrace: {e.StackTrace}");
```

Comment:
Why was this change necessary? This could should never execute if no App exists.

Reply:
This was to prevent log spam every frame if DeclarePreallocatedActors throws an exception.

Reply:
I don't see how App could ever be null. Now did you encounter that?